tests/aero.minova.rcp.uitests/pom.xml

```diff
@@ -50,7 +50,7 @@
                 <artifactId>tycho-surefire-plugin</artifactId>
                 <version>${tycho.version}</version>
                 <configuration>
-                    <argLine>${ui.test.vmargs}</argLine>
+                    <argLine>${additionalTestArgLine} -DbuildingWithTycho=true ${ui.test.vmargs}</argLine>
```

Comment:
Erklärungen hierzu in https://github.com/minova-afis/aero.minova.tycho.sample.multimodule